pegasus/forms/pd_workshop_survey.rb

```diff
@@ -34,27 +34,31 @@ def self.normalize(data)
       result[:how_much_learned_s] = required_enum data, :how_much_learned_s
       result[:how_motivating_s] = required_enum data, :how_motivating_s
 
-      result[:who_facilitated_ss] = required data[:who_facilitated_ss]
-
-      unless result[:who_facilitated_ss].class == FieldError
-        result[:how_clearly_presented_s] = {}
-        result[:how_interesting_s] = {}
-        result[:how_often_given_feedback_s] = {}
-        result[:help_quality_s] = {}
-        result[:how_comfortable_asking_questions_s] = {}
-        result[:how_often_taught_new_things_s] = {}
-        result[:things_facilitator_did_well_s] = {}
-        result[:things_facilitator_could_improve_s] = {}
-
-        result[:who_facilitated_ss].each do |facilitator|
-          result[:how_clearly_presented_s][facilitator] = required_enum(data, :how_clearly_presented_s, facilitator)
-          result[:how_interesting_s][facilitator] = required_enum(data, :how_interesting_s, facilitator)
-          result[:how_often_given_feedback_s][facilitator] = required_enum(data, :how_often_given_feedback_s, facilitator)
-          result[:help_quality_s][facilitator] = required_enum(data, :help_quality_s, facilitator)
-          result[:how_comfortable_asking_questions_s][facilitator] = required_enum(data, :how_comfortable_asking_questions_s, facilitator)
-          result[:how_often_taught_new_things_s][facilitator] = required_enum(data, :how_often_taught_new_things_s, facilitator)
-          result[:things_facilitator_did_well_s][facilitator] = stripped(data[:things_facilitator_did_well_s].try(:[], facilitator))
-          result[:things_facilitator_could_improve_s][facilitator] = stripped(data[:things_facilitator_could_improve_s].try(:[], facilitator))
+      enrollment = DASHBOARD_DB[:pd_enrollments].where(id: data[:enrollment_id_i]).first
+      no_expected_facilitators = DASHBOARD_DB[:pd_workshops_facilitators].where(pd_workshop_id: enrollment[:pd_workshop_id]).blank? if enrollment
```

Comment:
Perhaps a bit unusual, but it's not the [only](https://github.com/code-dot-org/code-dot-org/blob/67e02dbef181a3f4985c68c3602b37b8e36a52fe/pegasus/forms/volunteer_contact_2015.rb#L19-L19) db call in normalize.

I'm ok with this, but it might be better to do on the UI side and have a hidden field of the total number of facilitators along with the selected ones?

As it stands, the survey has these empty sections when there are no facilitators, which is a bit confuing:
![image](https://cloud.githubusercontent.com/assets/8405316/25106336/c4a7b914-237d-11e7-8693-5843a601284e.png)